src/indexClinicalData/transformToEsDonor.ts

```diff
@@ -32,8 +32,12 @@ export default (
 ): EsDonorDocument => {
   const submittedCoreDataPercent =
     mean(Object.values(mongoDoc?.completionStats?.coreCompletion || {})) || 0;
+  
+  const coreDataCompletedDate = submittedCoreDataPercent === 1
+    ? existingEsData?.coreDataCompletedDate || new Date(mongoDoc.updatedAt)
```

Comment:
not sure if I should use `updatedAt` or create a `new Date()` here.

also not sure if `existingEsData?.coreDataCompletedDate` could potentially exist.